kmir/src/kmir/convert_json/convert.py

```diff
@@ -7,8 +7,20 @@
     from collections.abc import Sequence, Mapping
     from pyk.kast.inner import KInner
 
+import hashlib
+
 from pyk.kast.inner import KApply, KSort, KToken
 
+func_map: dict[int, Mapping[str, str]] = {}
```

Comment:
Since this is a global, can we update its name to call attention to that fact? My preference is for SCREAMING_SNAKE_CASE, but other styles could work well too.

Reply:
I renamed both using this convention. I could try to thread these through from the top function to anywhere needed if you don't like the globals.

Reply:
I don't think that's necessary --- just a visual indicator is plenty.